starter-code/index.html

```diff
@@ -3,40 +3,59 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <div class = "header">
+    <img src="images/spotify-logo.png" alt="spotify-logo" id="spotify-logo">
+    <ul class = "nav">
+      <li>Premium</li>
+      <li>Discover</li>
+      <li>Help</li>
+      <li>Download</li>
+    </ul>
+
+  </div>
+  <div class = "content">
+    <div class = "music-for-everyone">
+        <h2>Music for everyone.</h2>
+        <p>Spotify is now free on mobile, tablet and computer.
+            Listen to the right music, wherever you are.</p>
+      </div>
+
+    <div class="whats-on-spotify">
```

Comment:
In my opinion the best way to organise this is to use a list, as the layout of those three blocks is the same, it will be easier to deal with responsiveness like this.

```html 
 <div id="head-main-content">
      <h2>What’s on Spotify?</h2>

      <ul>
        <li>
          <img src="images/music-icon.png">
          <h3>Millions of Songs</h3>
          <p> There are millions of
            <br>songs on Spotify</p>
        </li>
        <li>
          <img src="images/high-quality-icon.png">
          <h3>HD Music</h3>
          <p>Listen to music as if you
            <br> were listening live</p>
        </li>
        <li>
          <img src="images/devices-icon.png">
          <h3>Stream Everywhere</h3>
          <p>Stream music on your
            <br>smartphone, tablet or
            <br> computer
          </p>
        </li>
      </ul>
    </div>

```